src/Clients/FieldDefinitionsClient.cs

```diff
@@ -4,8 +4,14 @@
 
 namespace Laserfiche.Repository.Api.Client
 {
+    /// <summary>
+    /// The Laserfiche Repository Field Definitions API client.
+    /// </summary>
     partial interface IFieldDefinitionsClient
     {
+        /// <summary>
+        /// Get a collection of field definitions using paging. Page results are returned to the <paramref name="callback"/>.
```

Comment:
If you think it's better to use "Returns" instead of "Get", please consider this for the next cases also.

Reply:
I updated all the cases to use "Returns" instead of "Get"

Reply:
Thanks.